base/src/main/java/com/smartdevicelink/proxy/rpc/VideoStreamingCapability.java

```diff
@@ -49,6 +50,7 @@
 	public static final String KEY_DIAGONAL_SCREEN_SIZE = "diagonalScreenSize";
 	public static final String KEY_PIXEL_PER_INCH = "pixelPerInch";
 	public static final String KEY_SCALE = "scale";
+	public static final String KEY_ADDITIONAL_VIDEO_STREAMING_CAPABILITIES = "additionalVideoStreamingCapabilities";
```

Comment:
Based on the mobile_API.xml, the new parameter on line 53 needs the `/** @since 7.0 */` annotation.